san/get.py

```diff
@@ -9,9 +9,17 @@
     'ohlcv': 'get_ohlcv'
 }
 
+DEPRECATED_QUERIES_WARNING = [
+    'mvrv_ratio',
+    'nvt_ratio',
+    'realized_value',
+    'token_circulation'
+]
 
 def get(dataset, **kwargs):
     query, slug = parse_dataset(dataset)
+    if query in DEPRECATED_QUERIES_WARNING:
+        print('**NOTICE**\nThis metric will be deprecated in version 0.9.0, please use the get_metric equivalent instead')
```

Comment:
If the user executes queries in a loop he'll randomly see `This metric`. Add the metric name.

This warning is not understandable. The users of sanpy have zero knowledge of what `get_metric` is. The warning here should be ` ... mvrv_ratio will be deprecated ... please switch to `mvrv_usd` ... `

Reply:
Yeah, you're right, somehow mixed the things up, when I was writing them :D I will fix it in a bit